File: tools/dbench/answers/python_haiku/s1/f2_adults.py
with open("users.txt") as f:
    lines = f.readlines()
adults = []
for line in lines:
    parts = line.strip().split()
    if len(parts) == 3:
        try:
            age = int(parts[2])
            if age > 25:
                adults.append(parts[1])
        except ValueError:
            pass
print(" ".join(adults))

File: tools/dbench/answers/rust_haiku/s1/f2_adults.rs
use std::fs;

fn main() {
    let content = fs::read_to_string("users.txt").unwrap_or_default();
    let mut adults = Vec::new();

    for line in content.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i32>() {
                if age > 25 {
                    adults.push(parts[1].to_string());
                }
            }
        }
    }

    println!("{}", adults.join(" "));
}
